lib/game.rb

```diff
@@ -0,0 +1,101 @@
+require 'pry'
+class Game
+    
+    attr_accessor :board, :player_1, :player_2, :cells
+
+    WIN_COMBINATIONS = [[0,1,2],
+   [3,4,5],
+   [6,7,8],
+   [0,3,6],
+   [1,4,7],
+   [2,5,8],
+   [0,4,8],
+   [2,4,6]]
+
+
+   def initialize(player_1 = nil, player_2 = nil, board = nil)
+   #binding.pry
+   
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = Board.new
+    #how do I control which player goes first?
+
+    end
+
+
+    def current_player
+        board.turn_count.even? ? player_1 : player_2
+     end
+      
+     def over?
+        won? || draw?
+    end
+
+    def won?
+            WIN_COMBINATIONS.each do |win|
```

Comment:
Here you can use [`Enumerable#detect`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-detect) to return the winning combination or `nil` if none is found.

```ruby
  def won?
    WIN_COMBINATIONS.detect do |combo|
      @board.cells[combo[0]] == @board.cells[combo[1]] &&
      @board.cells[combo[0]] == @board.cells[combo[2]] &&
      @board.taken?(combo[0] + 1)
    end
  end
```